packages/swapper/src/swappers/ChainflipSwapper/endpoints.ts

```diff
@@ -277,6 +282,7 @@ export const chainflipApi: SwapperApi = {
       minimumPrice,
       refundAddress: from,
       commissionBps: serviceCommission,
+      boostFee: 0
```

Comment:
Also, do we have any way to test it yet? Until taproot sends are fixed

Reply:
> I'm missing something, why do we have boost fees for BTC but not for the other chains?
> 
> Because it's particularly slow using BTC?

Boost only applies to BTC. It's indeed invented because BTC is a slow beast, the others  have no such problems.
-----
No way to test BTC -> .... swaps until taproot is fixed no